README.md

```diff
@@ -13,33 +12,43 @@ by Harald Gruber-Vodicka, Elmar A. Pruesse, and Brandon Seah.
 
 ***NOTE*** Version 3 changed some input options and also how mapping-based taxa (NTUs) are handled. Please download the last release of v2.0 ([tar.gz archive](https://github.com/HRGV/phyloFlash/archive/v2.0-beta6.tar.gz)) for the old implementation. No changes have been made to the database setup, so databases prepared for v2.0 can still be used for v3.0.
 
-Quick-start
------------
+## Quick-start
 
-Download releases from the [releases](https://github.com/HRGV/phyloFlash/releases) page. If you clone the repository directly off GitHub you might end up with a version that is still under development.
+### Download via Conda
+
+[Conda](https://conda.io/docs/) is a package manager that will also install dependencies that are required if you don't have them already.
+
+phyloFlash is distributed through the [Bioconda](http://bioconda.github.io/) channel on Conda.
+
+```bash
+conda config --add channels bioconda # If you haven't set up Bioconda already
```

Comment:
should be 

```
conda config --add channels defaults
conda config --add channels bioconda
conda config --add channels conda-forge
```

The docs for the package are here:  http://bioconda.github.io/recipes/phyloflash/README.html 
which links to here for installing bioconda: http://bioconda.github.io/index.html#set-up-channels

Reply:
Might also mention the docker container https://quay.io/repository/biocontainers/phyloflash for those using singularity to run their stuff.

Reply:
Thanks, fixed bioconda instructions in `README.md` and docs